src/model_instance.rs

```diff
@@ -90,25 +111,27 @@ impl ModelInstance {
             ffm_learning_rate: 0.5, // vw default
             minimum_learning_rate: 0.0, 
             bit_precision: 18,      // vw default
-            hash_mask: 0, // DEPRECATED, UNUSED
             power_t: 0.5,
             ffm_power_t: 0.5,
             add_constant_feature: true,
             feature_combo_descs: Vec::new(),
             ffm_fields: Vec::new(),
             ffm_k: 0,
             ffm_bit_precision: 18,
-            ffm_separate_vectors: false, // DEPRECATED, UNUSED
             fastmath: true,
 			ffm_initialization_type: String::from("default"),
```

Comment:
Indent here is off

Reply:
not sure why its showing like this but in the code indent seems fine

Reply:
Ok, let's address this systematically in a separate PR then.